packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -245,6 +245,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 	 * @param timeoutMs - time limit on making the connection
 	 * @param epochTracker - track epoch changes
 	 * @param socketReferenceKeyPrefix - (optional) prefix to isolate socket reuse cache
+	 * @param connectionId - connection ID for the connection
```

Comment:
nit (don't block merging before the release): indicate this is optional like the one before it does

Reply:
done